### PR TITLE
[StructuralMechanicsApplication] Clean deprecated macro for variable key

### PR DESCRIPTION
**Pull request by loumalouomega**
**Description**
Clean deprecated macro for variable keyy

**Changelog**
- Clean deprecated macro for variable key

### CODE REVIEW

File: applications/StructuralMechanicsApplication/custom_response_functions/adjoint_elements/adjoint_finite_difference_base_element.cpp [hunk 1]
```diff
@@ -261,25 +261,10 @@ int AdjointFiniteDifferencingBaseElement<TPrimalElement>::Check(const ProcessInf
...
-    {
-        KRATOS_CHECK_VARIABLE_KEY(ROTATION);
-        KRATOS_CHECK_VARIABLE_KEY(ADJOINT_ROTATION);
-    }
-
     // TODO generic way of doing these checks without checking the dofs..
 
     // Check dofs
-    for (IndexType i = 0; i < r_geom.size(); ++i)
-    {
```
Comment (rubenzorrilla) [inline applications/StructuralMechanicsApplication/custom_response_functions/adjoint_elements/adjoint_finite_difference_base_element.cpp:282]: Aesthetic

File: applications/StructuralMechanicsApplication/custom_response_functions/adjoint_elements/adjoint_finite_difference_base_element.cpp [hunk 2]
```diff
@@ -289,8 +274,7 @@ int AdjointFiniteDifferencingBaseElement<TPrimalElement>::Check(const ProcessInf
         KRATOS_CHECK_DOF_IN_NODE(ADJOINT_DISPLACEMENT_Y, r_node);
         KRATOS_CHECK_DOF_IN_NODE(ADJOINT_DISPLACEMENT_Z, r_node);
 
-        if(mHasRotationDofs)
-        {
```
Comment (rubenzorrilla) [inline applications/StructuralMechanicsApplication/custom_response_functions/adjoint_elements/adjoint_finite_difference_base_element.cpp:293]: Aesthetic

File: applications/StructuralMechanicsApplication/custom_response_functions/adjoint_elements/adjoint_finite_difference_truss_element_3D2N.cpp [hunk 3]
```diff
@@ -87,12 +86,6 @@ int AdjointFiniteDifferenceTrussElement<TPrimalElement>::Check(const ProcessInfo
 template <class TPrimalElement>
 void AdjointFiniteDifferenceTrussElement<TPrimalElement>::CheckVariables() const
 {
-    KRATOS_CHECK_VARIABLE_KEY(DISPLACEMENT);
-    KRATOS_CHECK_VARIABLE_KEY(VELOCITY);
-    KRATOS_CHECK_VARIABLE_KEY(ACCELERATION);
-    KRATOS_CHECK_VARIABLE_KEY(DENSITY);
-    KRATOS_CHECK_VARIABLE_KEY(CROSS_AREA);
-    KRATOS_CHECK_VARIABLE_KEY(ADJOINT_DISPLACEMENT);
 }
```
Comment (rubenzorrilla) [inline applications/StructuralMechanicsApplication/custom_response_functions/adjoint_elements/adjoint_finite_difference_truss_element_3D2N.cpp]: Couldn't the method be removed?

**Review by rubenzorrilla — COMMENTED** [review]
Always room for aesthetic changes :smile: . Not approving because of the `CheckVariables` adjoint method question.

File: applications/StructuralMechanicsApplication/custom_response_functions/adjoint_elements/adjoint_finite_difference_base_element.cpp (hunk 1 above)
Comment (loumalouomega) [reply applications/StructuralMechanicsApplication/custom_response_functions/adjoint_elements/adjoint_finite_difference_base_element.cpp:282]: Technically is style, not aesthetic

File: applications/StructuralMechanicsApplication/custom_response_functions/adjoint_elements/adjoint_finite_difference_base_element.cpp (hunk 2 above)
Comment (loumalouomega) [reply applications/StructuralMechanicsApplication/custom_response_functions/adjoint_elements/adjoint_finite_difference_base_element.cpp:293]: Idem

File: applications/StructuralMechanicsApplication/custom_response_functions/adjoint_elements/adjoint_finite_difference_truss_element_3D2N.cpp (hunk 3 above)
Comment (loumalouomega) [reply applications/StructuralMechanicsApplication/custom_response_functions/adjoint_elements/adjoint_finite_difference_truss_element_3D2N.cpp]: Can be, but is not my implementation

**Comment by loumalouomega** [comment]
> Always room for aesthetic changes smile . Not approving because of the `CheckVariables` adjoint method question.

Done

**Review by philbucher — APPROVED** [review]
Thx

`CheckVariables` is a leftover, can be safely removed

**Pull request merged**